config/locales/en.yml

```diff
@@ -214,12 +214,14 @@ en:
       one: 'Subject'
       other: 'Subjects'
     leadership_level: 'Leadership level'
-    job_count_without_search: 'There is %{count} job listed'
-    job_count_plural_without_search: 'There are %{count} jobs listed'
-    job_count: '%{count} job matches your search'
-    job_count_plural: '%{count} jobs match your search'
-    job_count_with_location_category: '%{count} job match near %{location}'
-    job_count_plural_with_location_category: '%{count} jobs match near %{location}'
+    job_count_without_search: There is <span class='govuk-!-font-weight-bold'>%{count}</span> job listed
+    job_count_plural_without_search: "There are <span class='govuk-!-font-weight-bold'>%{count}</span> jobs listed"
+    job_count_with_keyword: "<span class='govuk-!-font-weight-bold'>%{count}</span> job matches <span class='govuk-!-font-weight-bold text-wrap-apostrophe'>%{keyword}</span>"
+    job_count_plural_with_keyword: "<span class='govuk-!-font-weight-bold'>%{count}</span> jobs match <span class='govuk-!-font-weight-bold text-wrap-apostrophe'>%{keyword}</span>"
+    job_count_with_location: "<span class='govuk-!-font-weight-bold'>%{count}</span> job found near <span class='govuk-!-font-weight-bold text-capitalize text-wrap-apostrophe'>%{location}</span>"
+    job_count_plural_with_location: <span class="govuk-!-font-weight-bold">%{count}</span> jobs found near <span class="govuk-!-font-weight-bold text-capitalize text-wrap-apostrophe">%{location}</span>
```

Comment:
have a look at how to use
https://thoughtbot.com/blog/pluralizing-rails-i18n-translations
here you can do:
```ruby
job_count_with_location:
  one: ...
  other: ...
```

Reply:
i had a look at this and it doesnt seem to work as expected, especially with `_html` postfix. so i have structured the en.yml a bit more in line with this structurally and it does make it more readable